src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/hmi_notifications_test.cc

```diff
@@ -232,7 +232,7 @@ class HMICommandsNotificationsTest
   DataAccessor<am::ApplicationSet> applications_;
 
   NiceMock<event_engine_test::MockEventDispatcher> mock_event_dispatcher_;
-  NiceMock<policy_test::MockPolicyHandlerInterface> mock_policy_handler_;
+  // NiceMock<policy_test::MockPolicyHandlerInterface> mock_policy_handler_;
```

Comment:
@ValeriiMalkov try to avoid such commented code in your pull requests. Better remove this line at all if it is unused.